medical_pharmacy/models/medical_pharmacist.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, tools
+from odoo.modules import get_module_resource
+
+
+class MedicalPharmacist(models.Model):
+    _name = 'medical.pharmacist'
+    _description = 'Medical Pharmacist'
+    _inherit = 'medical.abstract.entity'
+
+    @api.model
+    def _create_vals(self, vals):
+        vals.update({
+            'is_company': False,
+            'customer': False,
+        })
+        return super(MedicalPharmacist, self)._create_vals(vals)
+
+    @api.model
+    def _get_default_image(self, vals):
+        res = super(MedicalPharmacist, self)._get_default_image(vals)
```

Comment:
I feel that many lines of function `_get_default_image` will be repeated for each and every type of entity. My main idea is to split _get_default_image:
 - one part with the repeating code in  `medical_abstract_entity.py`
 - one part with the entity specific code in  `_get_default_image_path` in each of the entity.py files. 

What do you think of?
- moving `def _get_default_image(self, vals)` to `medical_abstract_entity.py`
- in inherited class, only override `_get_default_image_path` (instead of overriding the complete `_get_default_image` - "default partner one triggers instead": good, so no need to have `_get_default_image_path` in `medical_abstract_entity.py`

If you don't see my point, it is probably because I am missing something. Then feel free to ask "Show me the code!" :wink: .